chat-plugins/polls.js

```diff
@@ -0,0 +1,102 @@
+/* Poll commands plugin
+ * This is a poll commands system that will allow a voice (or roomvoice) to host 
+ * polls in terms of starting, reminding
+ * by: panpawn
+ */
+
+function splint(target) {
```

Comment:
You only ever use this function in one place, and it's short, so you should inline it